### ISSUE TITLE
no-undef when exporting types

### ISSUE THREAD

**Issue by darklight9811** [denoland/deno_lint#607]
# How to reproduce

``` typescript
type modelActions = "";

export default modelActions;
```

# Error
![image](https://user-images.githubusercontent.com/20667774/106968142-0dedb600-6727-11eb-9161-7781bed0557c.png)

**Comment by magurotuna** [comment]
@darklight9811 Thanks for the bug report :) I'll fix it soon

**Comment by darklight9811** [comment]
@magurotuna, hey, just another question related to typescript. Shouldn't arguments that start with `_` be ignored by typescript? It is complaining that it has any type, but it doesn't matter anyway because I'm not using it.

**Comment by magurotuna** [comment]
@darklight9811 could you show me a specific code snippet? Also, note that deno_lint is independent of the type checker of TypeScript. So if you're talking about TypeScript's quirk, there may be nothing we can do.

**Comment by darklight9811** [comment]
@magurotuna, oh, so probably that is the case, but just to show the code snippet:

```
const toString = (v: unknown, _, [max]: [string?]) => {
	const format = `${v}`;

	if (max && parseInt(max) < format.length)
		return format.substring(0, parseInt(max));
		
	return format;
};
```

I'm using this in a propertyDescriptor to format incoming values, the second parameter is actually the entire object.

**Comment by magurotuna** [comment]
@darklight9811 Yeah, unfortunately it looks like we can't do anything with deno_lint to suppress the warning message you get with your code.

**Comment by darklight9811** [comment]
@magurotuna , I would like to thank you for your patience, I've been encountering some pretty (censored word) here on github and it's a relief to find someone like you.
